package.json

```diff
@@ -45,11 +45,11 @@
     "release": "standard-version"
   },
   "devDependencies": {
-    "@capacitor/android": "3.4.0",
-    "@capacitor/cli": "3.4.0",
-    "@capacitor/core": "3.4.0",
+    "@capacitor/android": "^4.0.1",
```

Comment:
For thinking the future, I think Capacitor versions should not be fixed except for major versions.

Reply:
> Unfortunately, this PR is not yet ready for merging (e.g. there is a changelog for version 4, but we are only at version 1.3). Reviews and PR are welcome.
> 
> I recommend to use the **[Capacitor Firebase](https://github.com/capawesome-team/capacitor-firebase)** plugins, because it makes sure that the versions (native SDKs) of the plugins are compatible with each other. It also points out special details, like extra configuration steps when using Firebase Authentication with Firebase Messaging.

Does that mean this plugin won't be maintained and we shall move to capawesome plugin instead ?